src/pages/themes/submit/index.astro

```diff
@@ -64,7 +63,7 @@ const errorResponse = (formData: FormData, errors: z.typeToFlattenedError<typeof
 }
 
 if (Astro.request.method === "POST") {
-	// convert astro request to a node-fetch request
+	// convert astro request to a fetch request
 	// hopefully we won't have to do this in the future 🥲
 	const formData = await new Request(Astro.request.url, {
 		method: Astro.request.method,
```

Comment:
This makes me nervous! 😂 Does theme submission still work as expected?

Reply:
Yikes, yeah I didn't catch that when I was cleaning up the `node-fetch` uses!  I'm going to roll that back, the GitHub Action config is set to use Node v18 so there isn't really a need to switch the main project off `node-fetch`

Reply:
Reverted back to `node-fetch`, the Github Action config will handle installing node 18 and using built-in `fetch()` separate from the main project